problems/variables.js

```diff
@@ -31,7 +44,11 @@
 //  * Convert it to fahrenheit and output "`tempInCelsius`°C is `tempInFahrenheit`°F".
 //  * Now store a fahrenheit temperature into a variable.
 //  * Convert it to celsius and output "`tempInFahrenheit`°F is `tempInCelsius`°C."
-
+// var tempInCelsius = 32;
+// var tempInFahrenheit = (32 * 9/5) + 32;
```

Comment:
Try using the 'tempInCelsius' variable when writing the Celsius and Fahrenheit conversion code on line 48 and 50 as opposed to hard-coding the temperature you want to convert.